static/docs/commands-reference/pipeline_show.md

```diff
@@ -116,7 +116,7 @@ instead of stages.
                 `--------------'
   ```
 
-  To navigate, use `W`, `A`, `S`, `D` keys.
+  To navigate, use arrows or `W`, `A`, `S`, `D` keys.
```

Comment:
this note should be also part of the description and part of the `ascii` option above.

Reply:
after adding it to the `ascii` option.

![pipeline show](https://user-images.githubusercontent.com/16274905/61919947-6ca07400-af75-11e9-9fc5-b2c4e5c1fca8.PNG)